Services/CO.CDP.EntityVerification/Services/QueueBackgroundService.cs

```diff
@@ -0,0 +1,72 @@
+using Amazon.Runtime;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+
+namespace CO.CDP.EntityVerification.Services;
+
+public class QueueBackgroundService : BackgroundService
+{
+    private readonly IRequestListener _listener;
+    private readonly IConfiguration _config;
+    private readonly ILogger<IRequestListener> _logger;
+
+    public delegate void EvEventHandler(Message msg);
+    public event EvEventHandler? OnNewMessage;
+
+    public QueueBackgroundService(IRequestListener listener,
+        IConfiguration config,
+        ILogger<IRequestListener> logger)
+    {
+        _listener = listener;
+        _config = config;
+        _logger = logger;
+    }
+
+    protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+    { 
+        _listener.Register(this);
+        
+        var config = new AmazonSQSConfig
+        {
+            ServiceURL = _config["ServiceURL"],
+            UseHttp = false,
+            AuthenticationRegion = _config["AuthenticationRegion"]
+        };
+
+        // TODO: replace credentials with correct implementation.
```

Comment:
TODO... so we're not ready to merge? ;)

Reply:
Have moved accessKey and secretKey into appsettings